docs/_source/practical_guides/collect_responses.md

```diff
@@ -120,6 +120,83 @@ Once you have unified your responses, you will have a dataset that's ready for [
+agreement_metrics = metric.compute("alpha")
+# >>> agreement_metrics
+# [AgreementMetricResult(metric_name='alpha', result=0.0)]
+```
+
+Currently only [Krippendorf's alpha](https://en.wikipedia.org/wiki/Krippendorff%27s_alpha) is defined, for all the questions types except the `TextQuestion`. It applies to any number of annotators, with possible missing responses from them.
+The value from this measure is in the range [0,1] and is usually interpreted in the following way: alpha >= 0.8 indicates a reliable annotation, alpha >= 0.667 allows making tentative conclusions, while the lower values suggest the unreliable annotation.
+
+```{note}
+In case you want to dig deeper into the measurement of the agreement between different annotators, take a look at [*Implementations of inter-annotator agreement coefficients surveyed by Artstein
+and Poesio (2007), Inter-Coder Agreement for Computational Linguistics*](https://www.researchgate.net/publication/200044186_Inter-Coder_Agreement_for_Computational_Linguistics).
+```
+
+##### Supported metrics
+
+We plan on adding more support for other metrics so feel free to reach out on our Slack or GitHub to help us prioritize each task.
+
+| Question type/Metric  | alpha  |
+|:----------------------|:-------|
+| LabelQuestion         | ✔️      |
+| MultiLabelQuestion    | ✔️      |
+| RatingQuestion        | ✔️      |
+| RankingQuestion       | ✔️      |
+| TextQuestion          |        |
+
+
+#### Annotator metrics
+
+To evaluate whether the suggestions are useful or not.
+
+```python
+import argilla as rg
+from argilla.client.feedback.metrics import AnnotatorMetric
+
+feedback_dataset = rg.FeedbackDataset.from_argilla("...", workspace="...")
+metric = AnnotatorMetric(dataset=feedback_dataset, question_name="question_name")
+annotator_metrics = metric.compute("alpha")
+# >>> annotator_metrics
+# {'00000000-0000-0000-0000-000000000001': [AnnotatorMetricResult(metric_name='accuracy', result=0.5)], '00000000-0000-0000-0000-000000000002': [AnnotatorMetricResult(metric_name='accuracy', result=0.25)], '00000000-0000-0000-0000-000000000003': [AnnotatorMetricResult(metric_name='accuracy', result=0.5)]}
+```
+
+We would obtain a `dict` where the keys contain the `user_id` of a given annotator, and a list with the metrics requested. For the interpretation of these metrics we assume here that the *true* labels or responses corresponds to the points given by an annotator, and the *predictions* correspond to the suggestions given to a user. This way we can analyse whether we are offering good suggestions to a given annotator, and therefore simplifying it's work (shortening the annotation task), or otherwise we should change the suggestions given (or fine-tune our model if we can do it).
+
+##### Supported metrics
+
+We plan on adding more support for other metrics so feel free to reach out on our Slack or GitHub to help us prioritize each task.
```

Comment:
I think we should also provide default support for `MultiLabelQuestion`. I believe this is normally done using per-label metrics. https://medium.com/synthesio-engineering/precision-accuracy-and-f1-score-for-multi-label-classification-34ac6bdfb404

Reply:
Similarly for the `RankingQuestion` https://stats.stackexchange.com/questions/316522/making-a-measure-of-ranking-accuracy

Reply:
This one will be tackled in the `refactor` branch: https://github.com/argilla-io/argilla/pull/4271